lua/telescope/config.lua

```diff
@@ -113,6 +113,8 @@ function config.set_defaults(defaults)
   set("border", {})
   set("borderchars", { '─', '│', '─', '│', '╭', '╮', '╯', '╰'})
 
+  set("path_display", {})
```

Comment:
🤔 How can we validate this? @tjdevries. I failed to figure out a method of validation under the current structure. Any ideas.

I think validation should also be out of the scope of this PR